### PR TITLE
move to edition 2018 flavour: cargo fix --edition-idioms + cleanups

### PR DESCRIPTION
**Pull request by AnderEnder**
* apply `cargo fix --edition-idioms`
* remove `extern crate` and `macro_use` with `use`

### CODE REVIEW

File: tests/collapse_common/mod.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-extern crate inferno;
+use inferno;
```
Comment (jonhoo) [inline tests/collapse_common/mod.rs]: Huh, do we even need this `use`?

**Comment by jonhoo** [comment]
Ah, this is neat! Just that one minor question and then I'm happy to merge :D

File: tests/collapse_common/mod.rs (hunk 1 above)
Comment (AnderEnder) [reply tests/collapse_common/mod.rs]: I will recheck.

**Comment by AnderEnder** [comment]
and once more. missed pretty_assertions.

**Comment by jonhoo** [comment]
@AnderEnder looks like `cargo check` failed?

**Comment by AnderEnder** [comment]
it was fixed

**Pull request merged**